SPECS-EXTENDED/perl-Alien-Libxml2/perl-Alien-Libxml2.spec

```diff
@@ -0,0 +1,95 @@
+%global debug_package %{nil}
+Summary:        Install the C libxml2 library on your system
+Name:           perl-Alien-Libxml2
+Version:        0.14
+Release:        3%{?dist}
+License:        GPL+ OR Artistic
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Alien-Libxml2/
+Source0:        https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/Alien-Libxml2-%{version}.tar.gz#/perl-Alien-Libxml2-%{version}.tar.gz
+BuildRequires:  make
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+BuildRequires:  pkgconfig
+BuildRequires:  perl(:VERSION) >= 5.6
+BuildRequires:  perl(Alien::Base) >= 2.12
+BuildRequires:  perl(Alien::Build) >= 2.12
+BuildRequires:  perl(Alien::Build::MM) >= 2.12
+BuildRequires:  perl(Config)
+BuildRequires:  perl(ExtUtils::MakeMaker) >= 6.76
+BuildRequires:  perl(Test2::V0) >= 0.000060
+BuildRequires:  perl(Test::Alien)
+BuildRequires:  perl(Test::More)
+BuildRequires:  perl(alienfile)
+BuildRequires:  perl(base)
+BuildRequires:  perl(strict)
+BuildRequires:  perl(warnings)
+BuildRequires:  pkgconfig(libxml-2.0)
+Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
+Requires:       perl(Alien::Base) >= 2.12
+# This RPM package ensures libxml2 is installed on the system
+Requires:       pkgconfig(libxml-2.0) = %(type -p pkgconf >/dev/null && pkgconf --exists libxml-2.0 && pkg-config --modversion libxml-2.0 || echo 0)
+
+# Remove under-specified dependencies
+%global __requires_exclude %{?__requires_exclude:%{__requires_exclude}|}^perl\\(Alien::Base\\)$
```

Comment:
Let's try to keep macro definitions at the top of the spec, so they are easier to spot.